src/screens/LocationPage/utils.ts

```diff
@@ -22,7 +23,18 @@ export function getPageTitle(region: Region): string {
 
 export function getPageDescription(region: Region): string {
   const date = formatMetatagDate();
-  const { level: alarmLevel } = LocationSummariesByFIPS[region.fipsCode];
+  const { level: alarmLevel, metrics } = LocationSummariesByFIPS[
+    region.fipsCode
+  ];
   const levelInfo = LOCATION_SUMMARY_LEVELS[alarmLevel];
-  return `${date} COVID RISK LEVEL: ${levelInfo.detail(locationName(region))}`;
```

Comment:
This broke because `detail()` now returns a `ReactElement` containing a link but the description needs to be plain text.  But also, this description (which google shows in search results) ended up pretty brief, so I added our vaccination metric to it, which we're trying to highlight more anyway.